helm/cluster-azure/values.schema.json

```diff
@@ -514,6 +514,14 @@
                 "location": {
                     "type": "string",
                     "title": "Location",
+                    "$comment": "Valid options from https://github.com/giantswarm/capi-image-builder/blob/v1.6.10/helm/capi-image-builder/values.yaml#L48-L52",
```

Comment:
Can you maybe add a comment in that `capi-image-builder` file regarding updating `cluster-azure` when adding a new region?

Reply:
All right, added a draft PR here. https://github.com/giantswarm/capi-image-builder/pull/91